.github/workflows/sdk-ci/action.yml

```diff
@@ -0,0 +1,6 @@
+name: 'Continuous Integration for SDKs'
+runs:
+  using: 'composite'
+  steps:
+  - name: golang-ct
+    uses: ./.github/workflows/go-ct
```

Comment:
Can't the steps be defined here directly instead of having it in another file? I'd rather have it in one file

Reply:
In my original idea, I had planned to utilize various CI steps for different languages. here, it was to run go-lint (there is another upcoming PR) and then go-ct. Thus, I created a separate composite step to run these frequent CI steps. This should streamline the workflow and provide easier review.